05-script.md

```diff
@@ -208,9 +207,11 @@ to handle the case of parameters containing spaces
 Here's an example:
 
 ~~~ {.bash}
-$ cat sorted.sh
+$ nano sorted.sh
```

Comment:
There is no file `sorted.sh` yet, typing in `cat sorted.sh` does lead to an error for the learner. The style of showing a script's content with `cat` is often used in tutorials etc. online, but it is based on an understanding that the file does not exist on the reader's system, which is not clear for novice learners, I think.
The `nano ...` + `{.output}` (for file content) convention has been used earlier in the lesson.